test/test_cliptok.py

```diff
@@ -112,6 +112,12 @@ def test_tokenizer(self):
         self._run_tokenizer(["9 8 7 6 5 4 3 2 1 0"])
         self._run_tokenizer(["9 8 7 - 6 5 4 - 3 2 1 0"])
         self._run_tokenizer(["One Microsoft Way, Redmond, WA"])
+        self._run_tokenizer(["I can feel the magic, \n can you?"])
+        self._run_tokenizer(["\n Hey Cortana"])
+        self._run_tokenizer(["lower newer \n"])
+        self._run_tokenizer(["\n"])
+        self._run_tokenizer(["Testing multiple      sequences       of spaces"])
+        self._run_tokenizer(["      in the beginning and the end.      "])
```

Comment:
can you add a case of a single whitespace also?

Reply:
done!